storageservice/modules/blobs/types.bal

```diff
@@ -46,7 +46,18 @@ public type AccountInformationResult record {
     string skuName;
     string accountKind;
     string isHNSEnabled;
-    map<json> responseHeaders;
+    ResponseHeaders responseHeaders;
+};
+
+# Represents Response headers.
+#
+# + Date - A UTC date/time value that indicates the time at which the response was initiated  
+# + x\-ms\-version - Indicates the version of Blob Storage used to run the request 
+# + x\-ms\-request\-id - Uniquely identifies the request that was made, and can be used for troubleshooting
+public type ResponseHeaders record {
```

Comment:
If there are headers other than these defined, will they get mapped too as the record is `open`?

Reply:
Yes. I have added common headers for all the operations.
If there are any other headers, they also get added to this since its open